MAINTAINERS

```diff
@@ -1,3 +1,4 @@
+Alvin Lin <alvinlin123@gmail.com> (@alvinlin123)
```

Comment:
Up to Alvin, but I guess they want the company name too.

```suggestion
Alvin Lin, Amazon Web Services <alvinlin123@gmail.com> (@alvinlin123)
```

Reply:
Yes, they want the company name :)